plasmapy/physics/transport/collisions.py

```diff
@@ -78,16 +78,20 @@ def Coulomb_logarithm(T,
     UnitConversionError
         If the units on any of the inputs are incorrect.
 
-    UserWarning
-        If the input velocity is greater than 80% of the speed of
-        light.
-
-    TypeError
         If the n_e, T, or V are not Quantities.
 
     PhysicsError
         If the result is smaller than 1.
 
+    Warns
+    -----
+    ~astropy.units.UnitsWarning
+        If units are not provided and SI units are assumed
+
+    ~plasmapy.utils.RelativityWarning
+        If the input velocity is greater than 80% of the speed of
```

Comment:
I've got a more general question here. Is 80% a reasonable value? I think regime changes are typically defined at 1%.

Reply:
I can switch it to 1%, no problem. I also heard of that as the conventional value.

Reply:
My vote would be for 5%.  The Lorentz factor for 0.01c is ~1.00005, which would lead to discrepancies much less than measurement errors for pretty much any situation I can think of.  The Lorentz factor for 0.05c is ~1.001, which is when the errors are getting large enough that I would want to be notified.  On the other hand, if there are times when we need much higher precision, then I would be fine with a value of 1%.

Thanks for working on this!

Reply:
We can, if necessary, set a custom value for check relativistic whenever it pops up in transport or anywhere else, the decorator takes that as an argument. You guys probably know best.